Documentation/Stage-4-Maintenance-and-Security/Index.rst

```diff
@@ -53,9 +59,9 @@ Practical prerequisites
 
 Before you start this stage, please have the following things ready:
 
-* Item 1
-* Item 2
-* Item 3
+* A working TYPO3 installation under Git-based version control
+* A working continuous integration and continuous deployment workflow with local and remote parts
+* A contribution team with other contributors
```

Comment:
This may not be possible for all trainees. We may consider the trainer it-self as a contributor, but that is not necessary under the trainee control. I would remove that (or try to rephrase it in a different way, but I have no idea how)

Reply:
I think working with a team is important to know, but maybe it would be possible to simulate this through core contribution or something.